website/docs/docs/deploy/retry-jobs.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Retry your dbt jobs" 
+sidebar_label: "Retry jobs"
+description: "Rerun your errored jobs from start or the failure point."
+---
+
+If your dbt job run completed with a status of `result:error` , you can rerun it from start or from the point of failure in dbt Cloud.
+
+## Prerequisites
+
+- You have a [dbt Cloud account](https://www.getdbt.com/signup).
+- You must be using [dbt version](/docs/dbt-versions/upgrade-core-in-cloud) 1.6 or newer.
+- The more recent run of the job hasn't completed successfully. The latest status of the run is `error`.
+    - The job command that failed in the run must be one that supports the [retry command](/reference/commands/retry).
+
+## Rerun an errored job
+
+1. Select **Deploy** from the top navigation bar and choose **Run History.** 
+2. Choose the job run that has errored. 
+3. In the **Run Summary** tab on the job’s **Run** page, expand the run step that failed. An :x: denotes the failed step. 
+4. Examine the error message and determine how to fix it. After you have made your changes, save and commit them to your [Git repo](/docs/collaborate/git-version-control).
+5. Return to your job’s **Run** page. In the upper right corner, click **Rerun** and choose **Rerun from start** or **Rerun from failure**.
+    
+    If you chose to rerun from the failure point, a **Rerun failed steps** modal opens. It lists the run steps it will invoke: the failed step and any skipped steps. To confirm these run steps, click **Rerun from failure**. The job reruns from the failed command in the previously failed run. A banner at the top of the **Run Summary** tab captures this with the message, "This run resumed execution from last failed step".
```

Comment:
> It lists the run steps it will invoke:

Possibly confusing for non-English speakers only because the 1st "it" refers to the modal and the second "it" refers to dbt.

For the last sentence could use a little clarity in what the user should do. Should they check the banner for xyz success message? Is there a message they should worry about?

Reply:
- hmm, i am referring to the first modal here so it is the first "it". i was taught this was a valid use of "it". 
- nothing the user should do which is why it's phrased the way it is, just describing the banner and pointing it out the banner to the user